.prettierignore

```diff
@@ -10,4 +10,14 @@ build
 tests
 webdriver
 *.yml
-*.json
\ No newline at end of file
+*.json
+docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md
+docs/web-apps/automated-testing/cypress/yaml
+docs/web-apps/automated-testing/cypress/yaml.md
+docs/web-apps/automated-testing/playwright/yaml.md
+docs/web-apps/automated-testing/puppeteer/yaml.md
+docs/web-apps/automated-testing/replay/yaml.md
+docs/web-apps/automated-testing/testcafe/yaml.md
+docs/api-testing/on-prem
+docs/api-testing/composer
+docs/api-testing/composer.md
```

Comment:
@spider-sauce I think we should also add the following folders to .prettierignore --> for the [API reference](https://docs.saucelabs.com/dev/api/) and the docs folder for the platform integrations for Backtrace (error reporting).
```suggestion
docs/api-testing/composer.md
docs/dev/api
docs/error-reporting/platform-integrations
```

Reply:
I think by ignoring the embedded code languages I don't think this is necessary anymore. However we should still review/change anything in those files that needs to be updated before we merge.